Classes/Aggregate/BackendPreviewAggregate.php

```diff
@@ -180,9 +208,17 @@ protected function getProcessedData(array \$databaseRow, array \$processedTcaCol
                 continue;
             }
             \$processedRow[\$field] = [];
+            \$i = 0;
```

Comment:
$i is not needed here. You can access the result of $this->getProcessedData() if you store it in an own variable.